lib/workers/branch/index.ts

```diff
@@ -440,6 +440,29 @@ export async function processBranch(
     }
     const forcedManually = userRebaseRequested || !branchExists;
     config.forceCommit = forcedManually || branchPr?.isConflicted;
+
+    const stopUpdatingLabelPresent = branchPr?.labels?.includes(
+      config.stopUpdatingLabel
+    );
+    config.stopUpdatingLabelPresent = stopUpdatingLabelPresent;
+
+    const prRebaseChecked = branchPr?.body?.includes(
+      `- [x] <!-- rebase-check -->`
+    );
+
+    if (branchExists && dependencyDashboardCheck && stopUpdatingLabelPresent) {
```

Comment:
```suggestion
    if (branchExists && dependencyDashboardCheck && config.stopUpdatingLabelPresent) {
```

This probably will now fail prettier